<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MajorEssentialOption</class>
 <widget class="QMainWindow" name="MajorEssentialOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;
background-color: rgb(255, 208, 142);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_main_liberal_option_title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>전공필수</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_main_liberal_ment">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>139</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>이미 수강하신 전공필수를 선택하세요. </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right_arrow">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_right_arrow{
background-color: transparent;
border-image: url(:right_arrow3.png);
background: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>189</y>
      <width>291</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_2_0">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>자료구조</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2_1">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>컴퓨터구조</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2_2">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>알고리즘</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2_3">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>프로그래밍언어론</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_borderline">
    <property name="geometry">
     <rect>
      <x>-19</x>
      <y>-24</y>
      <width>889</width>
      <height>904</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/borderline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>630</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/5c06a53de64c4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_borderline</zorder>
   <zorder>label_main_liberal_option_title</zorder>
   <zorder>label_main_liberal_ment</zorder>
   <zorder>pushButton_right_arrow</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="menu_logo.qrc"/>
 </resources>
 <connections/>
</ui>
